src/mpu9150/mpu60x0.hpp

```diff
@@ -802,19 +802,20 @@ namespace upm {
     void getGyroscope(float *x, float *y, float *z);
 
 #if defined(SWIGJAVA) || defined(JAVACALLBACK)
-    /**
-     * get the accelerometer values
-     *
-     * @return Array containing X, Y, Z accelerometer values
-     */
-    float *getAccelerometer();
-
-    /**
-     * get the gyroscope values
-     *
-     * @return Array containing X, Y, Z gyroscope values
-     */
-    float *getGyroscope();
+// noel 2017/03/10 - Removed declarition since no implementation exists
+//    /**
+//     * get the accelerometer values
+//     *
+//     * @return Array containing X, Y, Z accelerometer values
+//     */
+//    float *getAccelerometer();
+//
+//    /**
+//     * get the gyroscope values
+//     *
+//     * @return Array containing X, Y, Z gyroscope values
+//     */
+//    float *getGyroscope();
 #endif
```

Comment:
Quite an oversight on my part.  We should put that on the list of things to do: fix the "Java Problem" once and for all like newer drivers do (use vectors, and have all java-specific method implementations present in the .hpp file only)